source/gui/addonStoreGui/views.py

```diff
@@ -0,0 +1,609 @@
+		wx.Panel.__init__(
+			self,
+			parent,
+			style=wx.TAB_TRAVERSAL | wx.BORDER_THEME
+		)
+
+		sizer = wx.BoxSizer(orient=wx.VERTICAL)
+		self.SetSizer(sizer)
+		self.contents = wx.BoxSizer(orient=wx.VERTICAL)
+		sizer.Add(
+			self.contents,
+			border=gui.guiHelper.BORDER_FOR_DIALOGS,
+			proportion=1,  # make vertically stretchable
+			flag=(
+				wx.EXPAND  # make horizontally stretchable
+				| wx.ALL  # and make border all around
+			),
+		)
+		# To make the text fields less ugly.
+		# See Windows explorer file properties dialog for an example.
+		self.SetBackgroundColour(wx.Colour("white"))
+
+		self.addonNameCtrl = wx.StaticText(
+			self,
+			label=AddonDetails._noAddonSelectedLabelText,
+			style=wx.ALIGN_CENTRE_HORIZONTAL | wx.ST_NO_AUTORESIZE
+		)
+		self._setAddonNameCtrlStyle()
+		self.contents.Add(self.addonNameCtrl, flag=wx.EXPAND)
+
+		self.contents.AddSpacer(gui.guiHelper.SPACE_BETWEEN_VERTICAL_DIALOG_ITEMS)
+
+		# It would be nice to override the name using wx.Accessible,
+		# but using it on a TextCtrl breaks the accessibility of the control entirely (all state/role is reset)
+		# Instead, add a hidden label for the textBox, Windows exposes this as the accessible name.
+		self.descriptionLabel = wx.StaticText(
+			self,
+			label=AddonDetails._descriptionLabelText
+		)
+		self.contents.Add(self.descriptionLabel, flag=wx.EXPAND)
+		self.descriptionLabel.Show(False)
+
+		self.descriptionTextCtrl = ExpandoTextCtrl(
+			self,
+			size=(self.scaleSize(400), -1),
+			style=(
+				0  # purely to allow subsequent items to line up.
+				| wx.TE_MULTILINE  # details will require multiple lines
+				| wx.TE_READONLY  # the details shouldn't be user editable
+				# Don't specify Auto-URL(wx.TE_AUTO_URL). No links expected in description, any URLs included
+				# in an add-on release metadata submission should not be easily activated.
+				| wx.BORDER_NONE
+			)
+		)
+		self.contents.Add(self.descriptionTextCtrl, flag=wx.EXPAND)
+		self.contents.Add(wx.StaticLine(self), flag=wx.EXPAND)
+
+		self._actionButtonMap: Dict[AddonActionVM, wx.Button] = {}
+		self._actionsSizer = self._createActionButtons()
+		self.contents.Add(self._actionsSizer)
+
+		self.contents.Add(wx.StaticLine(self), flag=wx.EXPAND)
+		self.contents.AddSpacer(gui.guiHelper.SPACE_BETWEEN_VERTICAL_DIALOG_ITEMS)
+
+		# It would be nice to override the name using wx.Accessible,
+		# but using it on a TextCtrl breaks the accessibility of the control entirely (all state/role is reset)
+		# Instead, add a hidden label for the textBox, Windows exposes this as the accessible name.
+		self.otherDetailsLabel = wx.StaticText(
+			self,
+			# Translators: Label for the text control containing extra details about the selected add-on.
+			# In the add-on store dialog.
+			label=pgettext("addonStore", "Other Details:")
+		)
+		self.contents.Add(self.otherDetailsLabel, flag=wx.EXPAND)
+		self.otherDetailsLabel.Show(False)
+		self.otherDetailsTextCtrl = wx.TextCtrl(

```

Comment:
It looks like using the `wx.TextCtrl` is causing several issues:
1. Failure to create links under Windows 7 as described above.
2. Links cannot be easily activated as mentioned in the known issues of the PR
3. The presentations of links in speech is pretty strange (there are some silent symbols before it when navigating character by character, and the url of the link seems to be reported twice).


Given these issues has it been considered to use either [wx.html.HtmlWindow](https://docs.wxpython.org/wx.html.HtmlWindow.html#wx-html-htmlwindow) or [wx.html2.WebView](https://docs.wxpython.org/wx.html2.WebView.html)?
These would be standard virtual buffers, with known working behavior for links activation and presentation. They behavior would also don't depend on the version of Windows in use, and styling should be significantly easier when done with CSS.
The only slight issue seems to be accessibility of these web views when focusing, but the [work around in this SO question](https://stackoverflow.com/questions/55345724/wx-html2-webview-and-screen-readers) should  help.

Reply:
Due to the accessibility issues with the HTML window controls and the minimal requirements for displaying the information, I chose the `wx.TextCtrl`. Converting this into a HTML window in the future might be reasonable, however security of the content will need to be carefully considered.

For now, it is much simpler to resolve the two minor issues found:
1. On old versions of Windows, the URLs won't be converted into links.
2. Activating links is simple but was considered a minor polish issue.
3. There will be a fix for this separately, there was an attempt already: [Ignore hidden text in richEdit controls accessed with ITextDocument #13618](https://github.com/nvaccess/nvda/pull/13618), however it was reverted.